src/Server/Models/Player.cs

```diff
@@ -2,11 +2,11 @@
 
 public record Player : BaseEntity
 {
-    public Player(string name, string position, string headShotUrl)
+    public Player(string name, string position)
     {
         Name = name;
         Position = position;
-        HeadShotUrl = headShotUrl;
+        HeadShotUrl = $"https://superflex.blob.core.windows.net/superflex-blob/Headshots/{@Name.Replace(' ', '-')}.png";
```

Comment:
Currently yeah but I am not entirely familiar with the repo to ensure I have made all required changes and that there isnt a better solution. If an image is missing in the blob storage it leaves the headshot gray, for example Ja'marr Chase currently. I have 723 headshots saved so far so I should have a majority if not all the fantasy relevant players.

I still need to apply caching to the images but if my understanding is right that will all be on the storage side.

This proposed change though would rely on me keeping the blob up to date so that's probably the biggest drawback on the Dynamo side. A solution that defaults to blob if exists and uses your saved URL's as a secondary may be a better solution.